cmd/clusterctl/configuration/ibmcloud/provider-component/user-data/ubuntu/templates/master-user-data.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+set -e
+set -x
+(
+KUBELET_VERSION={{ .Machine.Spec.Versions.Kubelet }}
+VERSION=v${KUBELET_VERSION}
+NAMESPACE={{ .Machine.ObjectMeta.Namespace }}
+MACHINE=$NAMESPACE
+MACHINE+="/"
+MACHINE+={{ .Machine.ObjectMeta.Name }}
+CONTROL_PLANE_VERSION={{ .Machine.Spec.Versions.ControlPlane }}
+CLUSTER_DNS_DOMAIN={{ .Cluster.Spec.ClusterNetwork.ServiceDomain }}
+POD_CIDR={{ .PodCIDR }}
+SERVICE_CIDR={{ .ServiceCIDR }}
+ARCH=amd64
+
+swapoff -a
+# disable swap in fstab
+sed -i.bak -r 's/(.+ swap .+)/#\1/' /etc/fstab
+
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+touch /etc/apt/sources.list.d/kubernetes.list
+sh -c 'echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list'
+apt-get update -y
+apt-get install -y \
+    prips
+
+# Getting master public ip 
+# TODO: is there any general way to get IP address in IBM Cloud?
+# e.g. Openstack: curl --fail -s http://169.254.169.254/2009-04-04/meta-data/local-ipv4
```

Comment:
a set of openstack stuffs in those files, will this be changed in this commit or you prefer to use next PR to handle?

Reply:
This is just an example to show that we may need the common way as `Openstack` to get host API. I will handle it if `TODO` is done or removed.